CHANGES

```diff
@@ -25,6 +25,7 @@ Enhancements
 
 Fixes
   - documented conda installation (available since 0.6.0) (#192)
+  - fixed how AMBER output parser reads time information (Issue #221, PR #219)
```

Comment:
AMBER parser now use ntpr to reads time information.

Reply:
Yeah, this sounds much better, changed it :-)